apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -212,7 +212,7 @@ public function fetch_vertical_data() {
 			$request_url = add_query_arg(
 				[ '_locale' => $this->get_iso_639_locale() ],
 				'https://public-api.wordpress.com/wpcom/v2/verticals/' . $vertical_id . '/templates'
-			);
+			);error_log($request_url);
```

Comment:
Was this accidentally included? :)

Reply:
Yes. I reverted it yesterday but forgot to push